charts/pulsar/templates/broker-statefulset.yaml

```diff
@@ -194,9 +195,14 @@ spec:
         - configMapRef:
             name: "{{ template "pulsar.fullname" . }}-{{ .Values.broker.component }}"
         volumeMounts:
-          {{- include "pulsar.broker.token.volumeMounts" . | nindent 10 }}
-          {{- include "pulsar.broker.certs.volumeMounts" . | nindent 10 }}
+          - mountPath: "/logconf"
+            name: "{{ template "pulsar.fullname" . }}-{{ .Values.broker.component }}-log4j-config"
+        {{- include "pulsar.broker.token.volumeMounts" . | nindent 10 }}
+        {{- include "pulsar.broker.certs.volumeMounts" . | nindent 10 }}
       volumes:
+      - name: "{{ template "pulsar.fullname" . }}-{{ .Values.broker.component }}-log4j-config"
```

Comment:
volume and volumeMount are required for different components. why not just create templates in `__helper.tpl`? Just like what we did for `pulsar.broker.token.volumes`?

Reply:
Good idea, will fix it